pkgs/games/corsix-th/corsix-th.nix

```
@@ -0,0 +1,72 @@
+{ baseName ? "corsix-th", version ? "trunk", audioSupport ? true, movieSupport ? true, freetypeSupport ? true, buildDocs ? false, enableUnitTests ? false, stdenv, cmake, lib, githubSource, SDL2, lua5_3, makeWrapper, SDL2_mixer, soundfont-fluid, fluidsynth, ffmpeg, freetype, doxygen, catch2 }:
+
+with lib;
+
+let
+  lua = {
+    env = lua5_3.withPackages (ps: with ps; [ lpeg luafilesystem ]);
+    packageDir = "${lua.env.outPath}/lib/lua/5.3";
+  };
+
+  SDL2_mixer_fluid =
+    if audioSupport then
+      SDL2_mixer.overrideAttrs
+        (old: rec {
+          configureFlags = old.configureFlags ++ [
+            " --enable-music-midi-fluidsynth-shared"
+            " --disable-music-midi-timidity"
+          ];
+        }) else null;
```

```suggestion
  SDL2_mixer_fluid = SDL2_mixer.overrideAttrs
    (old: rec {
      configureFlags = old.configureFlags ++ [
        " --enable-music-midi-fluidsynth-shared"
        " --disable-music-midi-timidity"
      ];
    });
```
This is not evaled when disabled.